source/command-monitoring/command-monitoring.rst

```diff
@@ -112,6 +112,35 @@ This specification defines the monitoring of inidividual commands and in that re
 an event for each command a bulk write executes. Each of these commands, however, must be linked
 together via the same {{operationId}}.
 
+---------
+Rationale
+---------
+
+The command listener API is responsible only for receiving and handling events sent from the lowest
```

Comment:
This prose only makes sense in the context of FAQ #2.  It doesn't stand on its own.